### PR TITLE
Feature signup mailer

### PR DESCRIPTION
**Pull request by christiangelone**
## Summary
 * Cover to send a welcome email when a user signs up

## Known Issues
* No Issues

## Trello Card
 * https://trello.com/c/7kQhSZ8H

### CODE REVIEW

File: docs/admin.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <html><head><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" /><script src="https://code.jquery.com/jquery-1.12.2.min.js"></script><script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script><style> pre { width: 95%; } </style></head><body><div class="container"><div class="jumbotron"><h1>Admin</h1></div><div class="panel panel-default"><div class="panel-heading" role="tab" id="heading-admin-users-POST"><h4 class="panel-title"><a role="button" data-toggle="collapse" data-parent="#accordion" aria-expanded="true" aria-controls="collapse-admin-users-POST" href="#collapse-admin-users-POST">POST /admin/users</a></h4></div><div id="collapse-admin-users-POST" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading-admin-users-POST"><div class="panel-body"><h4>Request Data</h4><div class="container"><h5>Headers</h5><pre>{
+<<<<<<< HEAD
```
Comment (gonziesc) [inline docs/admin.html]: conflicts

File: app/services/mailer/mailer.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const config = require('../../../config');
+const nodemailer = require('nodemailer');
+
+exports.send = (to, content) => {
+  const transporter = nodemailer.createTransport(config.common.mailer);
+  return transporter.sendMail({
+    from: `"Wolox Training 👤" <${config.common.mailer.auth.user}>`,
+    to,
+    subject: content.subject,
+    html: `<!DOCTYPE html>
```
Comment (gonziesc) [inline app/services/mailer/mailer.js]: usually we do this in a separate file

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/services/mailer/mailer.js (hunk 2 above)
Comment (christiangelone) [reply app/services/mailer/mailer.js]: Ok, i moved the layout as a new template & compose it with the signup template
No more hardcoded html in mailer

File: docs/admin.html (hunk 1 above)
Comment (christiangelone) [reply docs/admin.html]: solved!

**Review by gonziesc — APPROVED** [review]

**Pull request merged**